check-http/lib/check_http.go

```diff
@@ -30,9 +31,17 @@ type checkHTTPOpts struct {
 	Headers            []string `short:"H" description:"HTTP request headers"`
 	Regexp             string   `short:"p" long:"pattern" description:"Expected pattern in the content"`
 	MaxRedirects       int      `long:"max-redirects" description:"Maximum number of redirects followed" default:"10"`
+	Method             string   `short:"m" long:"method" choice:"GET" choice:"HEAD" choice:"POST" choice:"PUT" default:"GET" description:"Specify a GET, HEAD, POST, or PUT operation"`
 	ConnectTos         []string `long:"connect-to" value-name:"HOST1:PORT1:HOST2:PORT2" description:"Request to HOST2:PORT2 instead of HOST1:PORT1"`
 	Proxy              string   `short:"x" long:"proxy" value-name:"[PROTOCOL://][USER:PASS@]HOST[:PORT]" description:"Use the specified proxy. PROTOCOL's default is http, and PORT's default is 1080."`
-        BasicAuth          string   `long:"user" value-name:"USER[:PASSWORD]" description:"Basic Authentication user ID and an optional password."`
+	BasicAuth          string   `long:"user" value-name:"USER[:PASSWORD]" description:"Basic Authentication user ID and an optional password."`
+	RequireBytes       int64    `short:"B" long:"require-bytes" description:"Check the response contains exactly BYTES bytes" default:"-1"`
+	Body               string   `short:"d" long:"body" description:"Send a data body string with the request"`
+	MinBytes           int64    `short:"g" long:"min-bytes" description:"Check the response contains at least BYTES bytes" default:"-1"`
+	Timeout            int64    `short:"t" long:"timeout" description:"Set the total execution timeout in seconds" default:"15"`
```

Comment:
Is this default value comes from?

Basically, we shouldn't change current behavior so I suggest to change this value to zero, it means no timeout.
https://golang.org/pkg/net/http/#Client

Reply:
yes, you right, by defaut we should let 0.